browser/brave_ads/android/java/org/chromium/chrome/browser/dialogs/BraveAdsNotificationDialog.java

```diff
@@ -70,44 +81,73 @@ public static void showAdNotification(Context context, final String notification
         window.setFlags(WindowManager.LayoutParams.FLAG_NOT_TOUCH_MODAL,
```

Comment:
Do we still need this call given that flag had been set before?

Reply:
Will fix.